core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -459,14 +465,24 @@ class WorkflowResource {
   @Path("/search")
```

Comment:
Since this is essentially the same as `/list` when no query string parameter is provided, we should consolidate the code.

Reply:
I suggest we should get approval from prof Chen first before changing any code implemented by others.

Reply:
Maybe @shengquan-ni can chime in?